book/src/configuration.md

```diff
@@ -256,3 +257,38 @@ render = true
 character = "╎" # Some characters that work well: "▏", "┆", "┊", "⸽"
 skip-levels = 1
 ```
+
+### `[editor.soft-wrap]` Section
```

Comment:
Docs for soft wrap seems to have been included erroneously?

Reply:
Good catch ! Don't know how though x)